activities/value-and-impact/all-press.md

```diff
@@ -0,0 +1,31 @@
+---
+type: resource
+---
+
+
+# All press and media mentions of the Foundation for Public Code
+
+Here's the complete list of all mentions of the Foundation for Public Code on 3rd party platforms.
+
+We're always happy to work with journalists  - just email us at <press@publiccode.net>.
+
+You may also be interested in:
+
+- [a list of policy documents that mention the Foundation for Public Code](https://about.publiccode.net/activities/value-and-impact/policy-documents.html)
+- [our conference talks and articles we've written](https://projects.publiccode.net/talks-and-articles.html)
+
+## 2020
+
+- [Help: open source software will be mandatory, what now?](https://opensource.pleio.nl/news/view/7155b323-98f4-4ca0-9777-7a53ffbf2d16/help-open-source-software-wordt-verplicht-wat-nu), summary of a panel discussion with Boris van Hoytema, Ivonne Janssen-Dings and Koos Steenbergen at the Conferentie Nederland Digitaal in February 2021 (in Dutch)
```

Comment:
In this bullet point list there sometimes are a comma after the title and sometimes not. This should be consistent (but I don't know if we want them or not).

Reply:
Also dashes! I'll standardize.

Reply:
Also, this is not in 2020